podpac/core/node.py

```diff
@@ -504,9 +506,191 @@ def rem_cache(self, key, coordinates=None, mode=None):
         self.cache_ctrl.rem(self, key=key, coordinates=coordinates, mode=mode)
 
     # --------------------------------------------------------#
-    #  Class Methods
+    #  Class Methods (Deserialization)
     # --------------------------------------------------------#
 
+    @classmethod
+    def from_definition(cls, definition):
```

Comment:
I'm assuming this was mostly copied and pasted from the pipeline module? Or should I check this carefully?

Reply:
Yes, this is copied from the pipeline module. Same for the tests.